OneCommunityGlobal/HGNRest
# Olga _ updated error messages on login page

# Description
The error message on the login screen should be improved for clarity. 
When the username is not present in the system, it should display ""Username not found"" as the error message. 
Similarly, if the error is related to an incorrect password, the appropriate error message should be displayed.
Currently for both the scenarios the same error message is thrown “Invalid email and/ or password.”"

## Main changes explained:
Updated Logincontroller.js to include different error messages for non-existent user and for incorrect password.

## How to test:
1. check into current branch on the back end
2. do `npm run build` and `npm start` to run this PR locally
3. check into development branch on the front end
4. do `npm run start:local` to run the frontend
5. try to log in using an incorrect email (and any password). After clicking Submit button, you should see a message 'Username not found'
6. try to log in using a correct email and an incorrect password. After clicking Submit button, You should see a message 'Invalid password'.

## Screenshots or videos of changes:

Error message before:

<img width="613" alt="Before_errorMessage" src="https://github.com/OneCommunityGlobal/HGNRest/assets/107726913/381f27c9-e3ed-40dc-99e1-850e7c2b67bf">

Error messages for invalid email and incorrect password after:

Non existent email:
<img width="613" alt="After_emailField" src="https://github.com/OneCommunityGlobal/HGNRest/assets/107726913/d4088b22-10c5-4b12-8685-4fc39300cc6f">

Incorrect password:
<img width="613" alt="After_passwordField" src="https://github.com/OneCommunityGlobal/HGNRest/assets/107726913/19ead05c-664e-470a-8c69-1fb90418aa1e">
